src/app/header/product/product.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+  <div class="row">
+    <div class="col-2">
+      <nav class="nav flex-column">
+        <a class="nav-link active" aria-current="page">
+          <button (click)="getProductByCategory('Fruits & Vegetables')">
+            Fruits & Vegetables
```

Comment:
Please maintain the values in some variable/constant and loop over it

Reply:
done